### PR TITLE
SmugMug update followups for JBJSF

### PR DESCRIPTION
**Pull request by pushred**
The latest round of updates didn't address two outstanding to-do's on the Jon Bon Jovi Soul Foundation site: the "What's New" tab in the Photos section, and fuzzy image sizes on the Albums tab due to use of the thumbnail URL SmugMug provides, which is too small and the wrong aspect ratio in this instance.

Fixing the image size was easy, I just added some conditionals to use the small image size whenever that's been expanded. The recent images endpoint however ran into a blocker due to the implementation of paging for the display mode. All but one of the images returned by the [recent images endpoint][1] so far haven't been assigned to albums yet, except for the monster [Imported album][2] that the Clique Tools export goes into. Since that album is over 100 photos, the odds that a selected photo will be in the first 100 of that album are low. And it'd of course never be possible to request _all_ the photos.

Ideally we'd paginate within the recent images data rather than an album in this case. I could perhaps add a reserved static segment for the tags that does just that.

[1]: https://api.smugmug.com/api/v2/user/jonbonjovisoulfoundation!recentimages
[2]: https://api.smugmug.com/api/v2/album/KvzqDS

### CODE REVIEW

**Comment by pushred** [comment]
Woops, Joannic did [add recent images after all][1].

[1]: https://github.com/SparkartGroupInc/jungle-solidus/commit/ca01605760580b60d59ac29bffba48ca44924057

**Comment by pushred** [comment]
Alright that's more like it. Does the job, I'm happy. You happy @joanniclaborde?

File: widgets/utils/smugmug.js [hunk 1]
```diff
@@ -9,11 +9,19 @@ module.exports.albumsToTags = function(context, albums) {
...
+      ? album.Uris.HighlightImage.Image.Uris.ImageSizes.ImageSizes.SmallImageUrl
+      : album.Uris.HighlightImage.Image.ThumbnailUrl;
+  }
+
   return {
     tag:           album.Uri.split('/').pop(),
     title:         album.Title,
     description:   album.Description,
-    thumbnail_url: album.Uris.HighlightImage.Image ? album.Uris.HighlightImage.Image.ThumbnailUrl : null,
+    thumbnail_url: thumbnailImage || null,
```
Comment (joanniclaborde) [inline widgets/utils/smugmug.js]: The `|| null` is useless...

**Comment by joanniclaborde** [comment]
:+1:

**Pull request merged**